README.md

```diff
@@ -126,25 +126,15 @@ pip install -e .
 
 ### Linux
 
-First, you need to install protobuf.
-
-Ubuntu users: the quickest way to install protobuf is to run
+First, you need to install protobuf. The minimum Protobuf version required by ONNX is 3.0.0.
 
+Ubuntu 22.04 (and newer) users may choose to install protobuf via
```

Comment:
Since the minimum required version is 3.0.0, should this line be
```suggestion
Ubuntu 18.04 (and newer) users may choose to install protobuf via
```